app/views/coronavirus_landing_page/_page_header.html.erb

```diff
@@ -43,10 +46,10 @@
 
     <div class="govuk-grid-row covid__page-header-guidance-section">
       <div class="govuk-grid-column-two-thirds">
-        <p class="govuk-body govuk-font-colour-white govuk-!-margin-bottom-1">
+        <p class="govuk-body covid__page-header-body-pretext">
           <%= t("coronavirus_landing_page.guidance.pretext") %>
         </p>
-        <p class="govuk-body govuk-font-colour-white">
+        <p class="govuk-body">
           <a class="govuk-link covid__page-header-link" href="<%= t("coronavirus_landing_page.guidance.link.href") %>">
             <%= t("coronavirus_landing_page.guidance.link.text") %>
           </p>
```

Comment:
I think this trailing `</p>` is unnecessary - should be an `</a>`?

Reply:
This could be why the tests are all failing - just checking in a branch

Reply:
It was - that fix is merged if you rebase now

Reply:
omg! #fail